v3data/hypes/fees_yield.py

```diff
@@ -13,7 +13,7 @@
 
 
 class FeesYield(YieldData):
-    async def output(self, get_data=True):
+    async def get_feesYield_data(self, get_data=True):
```

Comment:
Renaming just for future convenience:  output could be each inherited class custom def .. like in this case, could be:

async def output(self, get_data=True):
      return get_feesYield_data(get_data)

Reply:
can we rename this to `get_fees_yield`? (All lowercase to follow convention)
I've am trying to distinguish the "data" classes (e.g. YieldData, those that query and collect the data) from the "calculations" class (e.g. FeesYield).